packages/core/src/readers/utils.ts

```diff
@@ -0,0 +1,105 @@
+// For LlamaParseReader.ts
+
+export const SupportedFileTypes: string[] = [
+  ".pdf",
```

Comment:
@hexapode is there a way to get this list of supported filetypes somehow dynamically?